SingularityUI/app/components/taskDetail/TaskFileBrowser.jsx

```diff
@@ -64,18 +70,33 @@ function TaskFileBrowser (props) {
     return _.sortBy(props.files, 'isDirectory').reverse();
   }
 
+  function recentlyModifiedTooltip() {
+    return <Tooltip id="tooltip">File is currently being written to</Tooltip>;
+  }
+
   return (
     <div>
       <Breadcrumbs items={pathItems} />
       <UITable
         data={getFiles() || []}
         keyGetter={(file) => file.name}
+        rowClassName={({mtime}) => { return isRecentlyModified(mtime) ? 'bg-info-light' : null; }}
         rowChunkSize={50}
         paginated={true}
         emptyTableMessage="No files exist in this directory"
         defaultSortBy="name"
+        striped={false}
       >
         <Column
+          className="icon-column"
+          label=""
+          id="icon"
+          key="icon"
+          cellData={(file) => isRecentlyModified(file.mtime) &&
```

Comment:
Because this depends on the current time, you should evaluate this once at the beginning of the render function instead of calling it 3 times.